File: examples/00-base/00-gettingStarted/plot.py
from plotz import *
from math import sin, pi

with Plot("plot") as p:
    p.title = r"My first \texttt{PlotZ} plot"
    p.x.label = "$x$"
    p.y.label = "$y$"

    p.plot(Function(sin, samples=50, range=(0, pi)),
           title=r"$\sin(x)$")

    p.legend("north east")

File: examples/00-base/00-gettingStarted/plot.jl
using PlotZ

Plot("essai") do p
    p.title = raw"My first \texttt{PlotZ} plot"
    p.x.label = raw"$x$"
    p.y.label = raw"$y$"

    plot!(p, PlotZ.Function(sin, samples=50, range=(0, pi)),
          title=raw"$\sin(x)$")
end
